CHANGELOG.md

```diff
@@ -139,9 +42,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed when Train loop config was run during `Trainer.predict` ([#6541](https://github.com/PyTorchLightning/pytorch-lightning/pull/6541))
```

Comment:
what happened to the 1.2.4 changelog? can you fix it in this PR?

Reply:
what do you mean, the three entries below were wrongly added before and some were duplicate, if you check the tree they are missing in v1.2.x